generator-fraudpreventiondata/generators/index.js

```diff
@@ -0,0 +1,21 @@
+var Generator = require('yeoman-generator');
+
+module.exports = class extends Generator {
```

Comment:
I wasn't able to test this, because I wasn't sure how to run the generator. Update the README contributing section with some instructions.

Reply:
👍